samples/zigbee/ncp/boards/nrf21540dk_nrf52840_usb.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
Please consider upgrading to the 2022 :slightly_smiling_face:

Reply:
adjusted